applications/user-api/src/main/java/no/fdk/userapi/UsersController.java

```diff
@@ -15,12 +13,10 @@
 public class UsersController {
     private static final Logger logger = LoggerFactory.getLogger(UsersController.class);
 
-    AltinnClient altinnClient;
-    AltinnUserConverter altinnUserConverter;
+    AltinnUserService altinnUserService;
```

Comment:
autowire

Reply:
siden sprint 4.3 trenger man ikke lenger autowired, det er implicit i construktor
https://www.baeldung.com/constructor-injection-in-spring

Reply:
Vel ja. Personlig er jeg fan av kode som er både kortere og mer forklarende, men hvis du vil ha det i constructor så er det lov.